### PR TITLE
[Compose] - 2626 - Show correct information in MessageListHeader subtitle

### PR DESCRIPTION
**Pull request by bychkovdmitry**
https://github.com/GetStream/stream-chat-android/issues/2626

### 🎯 Goal

Show either channel members for group channel or member presence for 1-to-1 conversation.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| photo_2021-11-15 17 01 27 | photo_2021-11-15 17 01 24 |

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt [hunk 1]
```diff
@@ -57,3 +59,47 @@ public fun Channel.getReadStatuses(userToIgnore: User?): List<Date> {
...
+ * @param context The context to load string resources.
+ * @param currentUser The currently logged in user.
+ * @return The text that represent the member status of the channel.
+ */
+public fun Channel.getMembersStatusText(context: Context, currentUser: User?): String {
+    val otherMembers = members.filter { it.user.id != currentUser?.id }
+
+    return when {
+        otherMembers.isEmpty() -> ""
+        isOneToOne(currentUser) -> {
```
Comment (filbabic) [inline stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt:91]: I think we can remove the {} as it's only ever going to be one statement!

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt [hunk 2]
```diff
@@ -57,3 +59,47 @@ public fun Channel.getReadStatuses(userToIgnore: User?): List<Date> {
...
+            otherMembers.first()
+                .user
+                .getLastSeenText(context)
+        }
+        else -> {
+            context.resources.getQuantityString(
+                R.plurals.stream_compose_channel_members,
+                otherMembers.count(),
```
Comment (filbabic) [inline stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt:99]: Why do we have to use `count()` twice here? Also, could we pull it out in a variable if we really need to fetch the number twice?

**Review by filbabic — CHANGES_REQUESTED** [review]
Two minor comments!

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt (hunk 1 above)
Comment (bychkovdmitry) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt]: Doesn't agree here as the readability suffers. Anyway, committed the change.

Comment (zsmb13) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt]: +1 on readability, I'd also have kept the branch, but it's fine either way, let's not get lost in the detail 😃

Comment (filbabic) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt]: We can totally keep the braces if you feel it's more readable, I felt differently from my PoV, but it's 2 against one 😄

**Review by filbabic — APPROVED** [review]

**Pull request merged**